diagnostic.md

```diff
@@ -43,8 +47,9 @@ weather = "sunny";
 weather === "sunny";
 ```
 
-What are the values of these expressions?  Explain your answers.
-
+== means equal value and === means equal value and equal type. Therefore, === is more specific.
+weather = "sunny" is true, but
+weather=== "sunny" is false, they are not the same thing.
```

Comment:
weather is set to the string of "sunny" to when you check for strict equality, does "sunny" === "sunny"? it would be true.